docs/docs/quick-start.md

```diff
@@ -13,6 +13,12 @@ This quick start is intended for intermediate to advanced developers. For a gent
 
 **Note**: this video uses `npx`, which is a tool to execute an npm package without first installing it. Running the command `npx gatsby new` is the same as running `gatsby new` after installing the gatsby-cli on your computer.
 
+### Install Node.js
```

Comment:
Same as above. I'd recommend adding a note about prerequisites and centralizing the install instructions in the Preparing Your Environment section: https://www.gatsbyjs.org/docs/preparing-your-environment/